docs/sources/getting-started/get-started-grafana-ms-sql-server.md

```diff
@@ -10,11 +10,7 @@ weight: 500
 
 # Get started with Grafana and MS SQL Server
 
-Microsoft SQL Server is a popular relational database management system that is widely used in development and production environments. This topic walks you through the steps to create a series of dashboards in Grafana to display metrics from a MS SQL Server database. You can also configure the MS SQL Server data source on a [Grafana Cloud](/docs/grafana-cloud/) instance without having to host Grafana yourself.
-
-{{< docs/shared "getting-started/first-step.md" >}}
-
-> **Note:** You must install Grafana 5.1+ in order to use the integrated MS SQL data source.
+Microsoft SQL Server is a popular relational database management system that is widely used in development and production environments. This topic walks you through the steps to create a series of dashboards in Grafana to display metrics from a MS SQL Server database.
```

Comment:
nit: I think it makes sense to have a single sentence on each line for better line-based diffing. I realize you are just removing a sentence so feel free to ignore this.